functionalDB/chapter.md

```diff
@@ -1123,16 +1130,16 @@ We've finally built all of the components we need to to build our user-facing qu
 
 Our journey started with a conception of a different kind of database, and ended with one that:
 
-* Supports ACI transactions (durability was lost when we decided to have the data stored in-memory) 
-* Supports "what-if" interactions
-* Answers time-related questions 
-* Handle simple datalog queries that are optimized by using indexes
-* provides APIs for graph queries
-* Introduces and implemented the notion of evolutionary queries
+* Supports ACI transactions (durability was lost when we decided to have the data stored in-memory).
+* Supports "what if" interactions.
+* Answers time-related questions.
+* Handles simple datalog queries that are optimized with indexes.
+* Provides APIs for graph queries.
+* Introduces and implements the notion of evolutionary queries.
 
 There are still many things that we could improve: We could add caching to several components to improve performance; support richer queries; and add real storage support to provide data durability, to name a few.
 
 However, our final product can do a great many things, and was implemented in 488 lines of Clojure source code, 73 of which are blank lines and 55 of which are docstrings. 
 
-Finally, there's one thing that is still missing - a Name. 
-The only sensible option for an in-memory, index optimized, query supporting, library developer friendly, time aware functional database implemented in 360 lines of Clojure code is CircleDB.

```

Comment:
implemented in 360 lines of Clojure code => using exactly 360 lines of Clojure code